docker/docker_image.go

```diff
@@ -131,7 +132,13 @@ func GetDigest(ctx context.Context, sys *types.SystemContext, ref types.ImageRef
 
 	path := fmt.Sprintf(manifestPath, reference.Path(dr.ref), tagOrDigest)
 	headers := map[string][]string{
-		"Accept": manifest.DefaultRequestedManifestMIMETypes,
+		"Accept": {
+			imgspecv1.MediaTypeImageManifest,
+			manifest.DockerV2Schema2MediaType,
+			manifest.DockerV2Schema1SignedMediaType,
+			manifest.DockerV2Schema1MediaType,
+			imgspecv1.MediaTypeImageIndex,
```

Comment:
(If this approach worked, which isn’t really the case, the list should not include the OCI Index type either.)

Reply:
https://github.com/distribution/distribution/blob/6affafd1f030087d88f88841bf66a8abe2bf4d24/registry/handlers/manifests.go#L114

Reply:
https://github.com/distribution/distribution/blob/6affafd1f030087d88f88841bf66a8abe2bf4d24/registry/handlers/manifests.go#L114